<?xml version="1.0" ?>

<launch>
  <arg name="robot_name" default="thormang3" />
  <arg name="gazebo" default="false" />

  <param name="gazebo"                   value="$(arg gazebo)"    type="bool" />
  <param name="gazebo_robot_name"        value="$(arg robot_name)" />

  <param name="offset_file_path"         value="$(find thormang3_manager)/config/offset.yaml" />
  <param name="robot_file_path"          value="$(find thormang3_manager)/config/THORMANG3.robot" />
  <param name="init_file_path"           value="$(find thormang3_manager)/config/dxl_init.yaml" />

  <param name="ft_data_path"             value="$(find thormang3_manager)/config/ft_data.yaml" />
  <param name="ft_calibration_data_path" value="$(find thormang3_manager)/config/ft_calibration_data.yaml" />
  <param name="wrist_ft_calibration_data_path" value="$(find thormang3_manager)/config/wrist_ft_calibration_data.yaml" />

  <param name="action_file_path"         value="$(find thormang3_action_module)/data/motion_4096.bin" />

  <!-- THORMANG3 Manager -->
  <node name="thormang3_manager" pkg="thormang3_manager" type="thormang3_manager" output="screen" />
</launch>
